### ISSUE TITLE
Automate and scale component finder

### ISSUE THREAD

**Issue by photodow** [carbon-design-system/devtools#7]
The first version of the component list and dependency identifier was manually entered into an object of classes and titles. This was ok, but it would greatly reduce time and effort if we could automate this somehow.

```javascript
import * as dotcomComps from '@carbon/ibmdotcom-react';

dotcomComps.forEach(...);
```

### some ideas
 - Can we grab the class name from the react object itself?
 - Can we work with teams to provide a list themselves, or at least export the class name?
 - Create a second build process that builds each component, grabs its class name, and sets it in a new `json` file.

### roadblocks
 - Some components won't render if data isn't provided.

**Comment by photodow** [comment]
Look at how Cloud & Cognitive solved this in #64 

https://github.com/carbon-design-system/devtools/pull/64/files#diff-e72bf23815bbd853a59e191d36e9f82cc027d33defd832c7ffd4c9ba657bf206R72-R91

**Comment by photodow** [comment]
got something going over in #92 

###### previous manual results
```
carbon: 51
cloudPal: 23
ibmdotcom: 66
security: 39

total: 179
```


###### new automated results
```
carbon: 132
ibmdotcom: 71
cloudCognitive: 45
security 50
cloudPal: 23

total: 321
```
